.github/workflows/auto-author-assign.yml

```diff
@@ -0,0 +1,15 @@
+# .github/workflows/auto-author-assign.yml
+name: Auto Author Assign
+
+on:
+  pull_request:
```

Comment:
Correct me if I'm wrong, but wouldn't it trigger this workflow at many events? I guess we could run it only for `opened` and maybe `reopened`